clients/intro_course_developer_component/src/introCourse/pages/SeatAssignment/components/SeatUploader/SeatPlanDialog.tsx

```diff
@@ -27,14 +26,15 @@ export const SeatPlanDialog = ({ seatPlan }: SeatPlanDialogProps): JSX.Element =
           View Seat Plan
         </Button>
       </DialogTrigger>
-      <DialogContent className='max-w-2xl max-h-[80vh] overflow-y-auto'>
-        <DialogHeader>
+      <DialogContent className='max-w-2xl max-h-[90vh] flex flex-col'>
+        {/* Sticky header */}
+        <DialogHeader className='sticky top-0 bg-white'>
           <DialogTitle>Current Seat Plan</DialogTitle>
           <DialogDescription>This seat plan contains {seatPlan.length} seats.</DialogDescription>
         </DialogHeader>
 
-        {/* Render list of seats */}
-        <div className='mt-4'>
+        {/* Scrollable content area */}
```

Comment:
Same here, comment can be removed
```suggestion
        <div className='flex-1 overflow-y-auto my-4'>
```

Reply:
done